src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -59,17 +52,8 @@
       </slot>
     </template>
     <template #tray="{ closeNav }">
-      <Hierarchy
-        :currentTopicTitle="title"
-        :isSymbolDeprecated="isSymbolDeprecated"
-        :isSymbolBeta="isSymbolBeta"
-        :parentTopicIdentifiers="hierarchyItems"
-        :currentTopicTags="currentTopicTags"
-        :references="references"
-      />
       <NavMenuItems
         class="nav-menu-settings"
-        :previousSiblingChildren="breadcrumbCount"
```

Comment:
Because `breadcrumbCount` was using the hierarchy items to make the calculation. But it's true that we were missing the first sibling children, so I added it back with a fixed "1" https://github.com/apple/swift-docc-render/commit/437d7554d823495fceec1a7787689f18ad197d97